src/analysis/calculus/uniform_limits_deriv.lean

```diff
@@ -410,45 +410,61 @@ end limits_of_derivatives
 
 section deriv
 
-/-! ### `deriv` versions of above theorems -/
+/-! ### `deriv` versions of above theorems
+
+In this section, we provide `deriv` equivalents of the `fderiv` lemmas in the previous section.
+The protected function `promote_deriv` provides the translation between derivatives and Fréchet
+derivatives
+-/
 
 variables {ι : Type*} {l : filter ι} [ne_bot l]
   {𝕜 : Type*} [is_R_or_C 𝕜]
   {G : Type*} [normed_add_comm_group G] [normed_space 𝕜 G]
   {f : ι → 𝕜 → G} {g : 𝕜 → G} {f' : ι → 𝕜 → G} {g' : 𝕜 → G}
   {x : 𝕜}
 
+/-- Transform a derivative into a Fréchet derivative -/
+protected def promote_deriv (f' : 𝕜 → G) : 𝕜 → (𝕜 →L[𝕜] G) :=
+λ z, (1 : 𝕜 →L[𝕜] 𝕜).smul_right (f' z)
+
+/-- Transform a sequence of derivatives into a sequence of Fréchet derivatives -/
+protected def promote_deriv_seq (f' : ι → 𝕜 → G) : ι → 𝕜 → (𝕜 →L[𝕜] G) :=
+λ n, λ z, (1 : 𝕜 →L[𝕜] 𝕜).smul_right (f' n z)
```

Comment:
I would be happier if you just inlined these definitions. `λ n z, (1 : 𝕜 →L[𝕜] 𝕜).smul_right (f' n z)` is a lot more descriptive than `promote_deriv_seq f'` (even if it is a bit of a mouthful.

Reply:
Sure!